docs/docs/docs/01-core/authentication/00-sessions-and-jwt.md

```diff
@@ -63,12 +63,20 @@ Optional request fields on login/register:
 
 Configuration:
 
-- `admin.auth.secret` — required JWT secret used by admin origin
+- `admin.auth.secret` — JWT secret for symmetric algorithms (HS256, HS384, HS512)
+- `admin.auth.sessions.options.algorithm` — JWT algorithm (default: 'HS256')
+- `admin.auth.sessions.options.privateKey` — Private key for asymmetric algorithms (RS256, RS512, ES256, etc.)
+- `admin.auth.sessions.options.publicKey` — Public key for asymmetric algorithms (RS256, RS512, ES256, etc.)
```

Comment:
I think we can delete this actually, and just specify that `options` is what is passed in to `jwt.sign` et al